crates/ruff_python_ast/src/generated.rs

```diff
@@ -2761,24 +2806,24 @@ impl ExceptHandler {
 }
 
 #[derive(Clone, Debug, PartialEq)]
-pub enum FStringElement {
-    Expression(crate::FStringExpressionElement),
-    Literal(crate::FStringLiteralElement),
+pub enum FTStringElement {
+    Expression(crate::FTStringInterpolatedElement),
```

Comment:
Should we rename this to `InterpolationElement` (with or without the `FT` prefix) because that's the terminology used throughout PEP750